luda-editor/luda-editor-client/src/app/editor/mod.rs

```diff
@@ -353,6 +363,14 @@ impl Editor {
     }
     fn on_change_sequence(&mut self) {
         let sequence = self.get_sequence().clone();
+        match &self.selected_clip_id {
+            Some(selected_clip_id) => {
+                if sequence.get_clip(&selected_clip_id).is_none() {
+                    self.deselect_clip();
+                }
+            }
+            None => {}
+        }
```

Comment:
removing dangling selected clip. Is it better to make a function instead of big match code?

Reply:
I believe it would be prettier to use a function.
and also It seems good to use `If ~ let` because none arm does nothing.

Reply:
I changed it in branch `feature/multi-select-move`, which must be the next PR. Please check it in next PR.
https://github.com/NamseEnt/namseent/blob/feature/multi-select-move/luda-editor/luda-editor-client/src/app/editor/mod.rs#L386